### PR TITLE
Update Timothy's short bio

### PR DESCRIPTION
**Pull request by tlkh**

### CODE REVIEW

**Comment by tlkh** [comment]
@jamestiotio @JinBean to merge this is ok

**Comment by jamestiotio** [comment]
@tlkh Sure, looks good! Will merge

File: index.html [hunk 1]
```diff
@@ -195,10 +195,11 @@ <h1 class="title has-text-centered name-title">Hum Qing Ze</h1>
                         </div>
 
                         <div class="column is-4 has-text-centered" data-aos="fade-left">
-                            <h1 class="title has-text-centered name-title">Timothy Liu Kai Hui</h1>
+                            <h1 class="title has-text-centered name-title">Timothy Liu Kaihui</h1>
                             <p class="seperator-text">--role--</p>
```
Comment (jamestiotio) [inline index.html:199]: @tlkh You can add your role here, if you want. If no ideas then can just delete this line.

**Review by jamestiotio — COMMENTED** [review]
Added comment on line 199 of index.html

**Pull request merged**